packages/boot/test/bootstrapTests/test-liquidation-concurrent-1.ts

```diff
@@ -0,0 +1,557 @@
+    },
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 17.48,
+      },
+    },
+  ],
+  reserve: {
+    allocations: {
+      STARS: 0.306349,
+    },
+    shortfall: 0,
+  },
+  vaultsSpec: [
+    {
+      locked: 3.373,
+    },
+    {
+      locked: 3.024,
+    },
+    {
+      locked: 2.792,
+    },
+  ],
+  vaultsActual: [
+    {
+      locked: 3.525747,
+    },
+    {
+      locked: 3.181519,
+    },
+    {
+      locked: 2.642185,
+    },
+  ],
+};
+
+const outcomes = {
+  ATOM: atomOutcome,
+  STARS: starsOutcome,
+};
+
+test.before(async t => {
+  t.context = await makeLiquidationTestContext(t);
+  await addSTARsCollateral(t);
+});
+test.after.always(t => {
+  return t.context.shutdown && t.context.shutdown();
+});
+
+// Reference: Flow 1 from https://github.com/Agoric/agoric-sdk/issues/7123
+const checkFlow1 = async (t: ExecutionContext<LiquidationTestContext>) => {
+  // fail if there are any unhandled rejections
+  process.on('unhandledRejection', (error: Error) => {
+    t.fail(error.message);
+  });
+
+  const {
+    advanceTimeBy,
+    advanceTimeTo,
+    agoricNamesRemotes,
+    check,
+    setupStartingState,
+    priceFeedDrivers,
+    readLatest,
+    walletFactoryDriver,
+  } = t.context;
+
+  const cases = [
+    { collateralBrandKey: 'ATOM', managerIndex: 0 },
+    { collateralBrandKey: 'STARS', managerIndex: 1 },
+  ];
+
+  const metricsPaths = cases.map(
+    ({ managerIndex }) =>
+      `published.vaultFactory.managers.manager${managerIndex}.metrics`,
+  );
+
+  const buyer = await walletFactoryDriver.provideSmartWallet('agoric1buyer');
+
+  const setupVault = async (
+    collateralBrandKey: string,
+    managerIndex: number,
+  ) => {
+    await setupStartingState({
+      collateralBrandKey,
+      managerIndex,
+      price: setups[collateralBrandKey].price.starting,
+    });
+
+    const minter =
+      await walletFactoryDriver.provideSmartWallet('agoric1minter');
+
+    for (let i = 0; i < setups[collateralBrandKey].vaults.length; i += 1) {
+      const offerId = `open-${collateralBrandKey}-vault${i}`;
+      await minter.executeOfferMaker(Offers.vaults.OpenVault, {
+        offerId,
+        collateralBrandKey,
+        wantMinted: setups[collateralBrandKey].vaults[i].ist,
+        giveCollateral: setups[collateralBrandKey].vaults[i].atom,
+      });
+      t.like(minter.getLatestUpdateRecord(), {
+        updated: 'offerStatus',
+        status: { id: offerId, numWantsSatisfied: 1 },
+      });
+    }
+
+    // Verify starting balances
+    for (let i = 0; i < setups[collateralBrandKey].vaults.length; i += 1) {
+      check.vaultNotification(managerIndex, i, {
+        debtSnapshot: {
+          debt: { value: scale6(setups[collateralBrandKey].vaults[i].debt) },
+        },
+        locked: { value: scale6(setups[collateralBrandKey].vaults[i].atom) },
+        vaultState: 'active',
+      });
+    }
+  };
+
+  const placeBid = async (collateralBrandKey: string) => {
+    await buyer.sendOffer(
```

Comment:
`async` superfluous. `sendOffer` returns a promise the promise that `placeBid` need to.
```suggestion
  const placeBid = (collateralBrandKey: string) => buyer.sendOffer(
```

Reply:
Also please define this closer to where it's used. Consider inlining it since it's used only there.

I see the value of having a single function call for placing bids, but I think it would be clearer if it were `placeNewBids()`, matching the spec terminology